aws/data_source_aws_partition.go

```diff
@@ -16,6 +16,10 @@ func dataSourceAwsPartition() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"url_suffix": {
```

Comment:
Can you please name this `dns_suffix`? We will likely be introducing this as an argument to the provider configuration to override all endpoints and the "URL" naming could be potentially confusing to operators since URLs can contain HTTP-like paths as a suffix.

```suggestion
			"dns_suffix": {
```

Reply:
done